### PR TITLE
update(mini-css-extract-plugin): v2.3

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- `runtime` config option

https://github.com/webpack-contrib/mini-css-extract-plugin/releases/tag/v2.3.0

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by JounQin — APPROVED** [review]
Thank you, a weird thing is that it is said option name is `runtime` in its release log but actually it should be `noRuntime`. 😂

File: types/mini-css-extract-plugin/index.d.ts [hunk 1]
```diff
@@ -76,6 +76,13 @@ declare namespace MiniCssExtractPlugin {
          * @default 'text/css'
          */
         linkType?: string | false | 'text/css' | undefined;
+        /**
+         * An option to avoid the runtime generation. Assets are still extracted and can be used for a custom loading methods.
+         * For example, you can use [assets-webpack-plugin](https://github.com/ztoben/assets-webpack-plugin) to retreive them
+         * then use your own runtime code to download assets when needed.
+         * @default false
+         */
+        noRuntime?: boolean | undefined;
```
Comment (JounQin) [inline types/mini-css-extract-plugin/index.d.ts:85]: I checked the source code, it *is* `runtime` actually.

**Review by JounQin — CHANGES_REQUESTED** [review]

File: types/mini-css-extract-plugin/index.d.ts (hunk 1 above)
Comment (JounQin) [reply types/mini-css-extract-plugin/index.d.ts:85]: See https://github.com/webpack-contrib/mini-css-extract-plugin/commit/5cfb0abd96c223780aec755b1f642c8f8de30a1e

Comment (peterblazejewicz) [inline types/mini-css-extract-plugin/index.d.ts:85]:
```suggestion
        runtime?: boolean | undefined;
```

File: types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts [hunk 2]
```diff
@@ -121,4 +121,15 @@ configuration = {
     ],
 };
 
+{
+    // noRuntime
+    new MiniCssExtractPlugin({});
+    new MiniCssExtractPlugin({
+        noRuntime: false,
```
Comment (peterblazejewicz) [inline types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts:128]:
```suggestion
        runtime: false,
```

File: types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts [hunk 3]
```diff
@@ -121,4 +121,15 @@ configuration = {
...
+    });
+    new MiniCssExtractPlugin({
+        noRuntime: true,
```
Comment (peterblazejewicz) [inline types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts:131]:
```suggestion
        runtime: true,
```

File: types/mini-css-extract-plugin/index.d.ts (hunk 1 above)
Comment (peterblazejewicz) [reply types/mini-css-extract-plugin/index.d.ts]: thx! amended!

File: types/mini-css-extract-plugin/index.d.ts [hunk 4]
```diff
@@ -76,6 +76,13 @@ declare namespace MiniCssExtractPlugin {
          * @default 'text/css'
          */
         linkType?: string | false | 'text/css' | undefined;
+        /**
+         * An option to avoid the runtime generation. Assets are still extracted and can be used for a custom loading methods.
+         * For example, you can use [assets-webpack-plugin](https://github.com/ztoben/assets-webpack-plugin) to retreive them
+         * then use your own runtime code to download assets when needed.
+         * @default false
```
Comment (JounQin) [inline types/mini-css-extract-plugin/index.d.ts:83]:
```suggestion
         * @default true
```

File: types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts [hunk 5]
```diff
@@ -121,4 +121,15 @@ configuration = {
     ],
 };
 
+{
+    // noRuntime
```
Comment (JounQin) [inline types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts:125]:
```suggestion
    // runtime
```

File: types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts (hunk 2 above)
Comment (JounQin) [inline types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts:128]:
```suggestion
        runtime: false,
```

File: types/mini-css-extract-plugin/index.d.ts [hunk 6]
```diff
@@ -76,6 +76,13 @@ declare namespace MiniCssExtractPlugin {
          * @default 'text/css'
          */
         linkType?: string | false | 'text/css' | undefined;
+        /**
+         * An option to avoid the runtime generation. Assets are still extracted and can be used for a custom loading methods.
+         * For example, you can use [assets-webpack-plugin](https://github.com/ztoben/assets-webpack-plugin) to retreive them
+         * then use your own runtime code to download assets when needed.
```
Comment (JounQin) [inline types/mini-css-extract-plugin/index.d.ts:82]:
```suggestion
         * Allows to enable/disable the runtime generation.
         * CSS will be still extracted and can be used for a custom loading methods.
         * For example, you can use [assets-webpack-plugin](https://github.com/ztoben/assets-webpack-plugin) to retreive them then use your own runtime code to download assets when needed.
```

File: types/mini-css-extract-plugin/index.d.ts (hunk 4 above)
Comment (peterblazejewicz) [reply types/mini-css-extract-plugin/index.d.ts]: sorry, I've misread the changes, amended

**Review by JounQin — APPROVED** [review]

**Comment by JounQin** [comment]
@peterblazejewicz The PR description should be updated too.

**Comment by peterblazejewicz** [comment]
Ready to merge 💘

**Pull request merged**